open-company/open-company-email
# Sparklines for Finance and Growth

This PR adds sparklines into the finance and growth sections of emails, and removes the "card" styling in favor of something more minimal.

https://trello.com/c/pBN9SNtb
https://trello.com/c/VzhV4x8T

Depends on this commit to infra (already merged): https://github.com/belucid/open-company-infrastructure/commit/5e94c9648ac639892001d37190f0692b58acf91a

To test:
- Review the code
- Run through the snapshots in `content.clj`'s REPL comment section, looking at `hiccup.html` after each one. Make changes to the JSON between runs, are they reflected in the sparklines? Try to challenge the sparkline code w/ challenging data changes, any problems?
- Send some email updates from staging (already deployed). Check the email in mobile and web email viewers.
- Merge.
- Deploy to beta.
- Upgrade your data plan.
